modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -38,15 +32,8 @@ Use `gadmin backup` to xref:tigergraph-server:backup-and-restore:backup-cluster.
 
 For how to restore a cluster from a backup that was created from another database cluster (cross cluster), refer to xref:tigergraph-server:backup-and-restore:cross-cluster-backup.adoc[].
 
-If you are setting up both the primary cluster and the DR cluster from scratch, you can skip Steps 1, 2, and 4 and only perform Step 3.
-
```

Comment:
We recommend the same process for enabling PR and DR clusters, whether they are built from scratch or not. So deleting this section here.